server/src/routes/serializers/ReviewSerializer.js

```diff
@@ -0,0 +1,17 @@
+import UserSerializer from "./UserSerializer.js"
+
+class ReviewSerializer {
+  static async getSummary(review) {
+    const allowedAttributes = ["id", "comment", "rating"]
+
+    let serializedReview = {}
+    for (const attribute of allowedAttributes) {
+      serializedReview[attribute] = review[attribute]
+    }
+    const relatedUser = await review.$relatedQuery("users")
+    const serializedReviews = relatedUser.map((user) => UserSerializer.getSummary(user))
```

Comment:
Since it's just a single `user` and no long an array of users we won't need .map.

Something like,
```js
const serializedUser = UserSerializer.getSummary(relatedUser)
```